### DISCUSSION TITLE
Support for raspberry zero w

### DISCUSSION THREAD

**Discussion by DrRek** [tiny-pilot/tinypilot#395] [Ideas]
Hi @mtlynch 
While looking for a KVM device to use in my home setup when I'm traveling I found this project and it looks exactly like the perfect solution I was searching for.
I would be interested into trying this out and I was wondering if it was possible to adapt the project to work with a raspberry zero (since is the only device I have available during these holiday's days :laughing: )
Do you think it would be possible to:
- use the wireless connection of the pi zero instead of the ethernet port of the pi 4
- use the usb OTG port for the keyboard
- use the usb power to power the pi zero (with an external power suply, not connected to the controlled pc)
- use the hdmi port to connect to the pc hdmi port to act as a monitor instead of the usb explained in your project.

I would be more than willing  to test this edit myself, but first, since I don't have that much experience with raspberry I would like to know your opinion first. In your experience do you think I'm ignoring something that won't allow me to use the pi zero? (e.g. maybe the HDMI port on the zero can't be used as input idk)

**Comment by mtlynch** [comment]
Thanks for reaching out!

It does work on the Pi Zero W, but it's pretty slow. You can see some images and videos in this Twitter thread:

https://twitter.com/tinypilotkvm/status/1329881175847002113

You can see some additional discussion about users experimenting with TinyPilot on Pi Zero W in #93.

**Reply by DrRek** [reply]
Cool, thanks
